docs/core/deploying/trimming-options.md

```diff
@@ -9,27 +9,47 @@ ms.date: 08/25/2020
 
 The following MSBuild properties and items influence the behavior of [trimmed self-contained deployments](trim-self-contained.md). Some of the options mention `ILLink`, which is the name of the underlying tool that implements trimming. More information about the underlying tool can be found at the [Linker documentation](https://github.com/mono/linker/tree/master/docs).
 
+Trimming with `PublishTrimmed` was introduced in .NET Core 3.0. The other options are available only in .NET 5 and above.
+
 ## Enable trimming
 
 - `<PublishTrimmed>true</PublishTrimmed>`
 
    Enable trimming during publish, with the default settings defined by the SDK.
 
-When using `Microsoft.NET.Sdk`, this will perform assembly-level trimming of the framework assemblies from the netcoreapp runtime pack. Application code and non-framework libraries are not trimmed. Other SDKs may define different defaults.
+When using `Microsoft.NET.Sdk`, this will trim the framework assemblies from the netcoreapp runtime pack. Application code and non-framework libraries are not trimmed. Other SDKs may define different defaults.
 
 ## Trimming granularity
 
 The following granularity settings control how aggressively unused IL is discarded. This can be set as a property, or as metadata on an [individual assembly](#trimmed-assemblies).
 
+- `<TrimMode>link</TrimMode>`
+
+    Enable member-level trimming, which removes unused members from types. This is the default in .NET 6+.
+
 - `<TrimMode>copyused</TrimMode>`
 
    Enable assembly-level trimming, which will keep an entire assembly if any part of it is used (in a statically understood way).
 
-- `<TrimMode>link</TrimMode>`
+Assemblies with `<IsTrimmable>true</IsTrimmable>` metadata but no explicit `TrimMode` will use the global `TrimMode`. The default `TrimMode` for `Microsoft.NET.Sdk` is `link` in .NET 6+, and `copyused` in previous versions.
```

Comment:
> Assemblies with `<IsTrimmable>true</IsTrimmable>` metadata

What exactly does this mean? What assemblies? `TrimmableAssembly`? `ManagedAssemblyToLink`?

Reply:
Clarified this in the section before the `TrimMode` bullets